IBM/ibm-common-service-operator
# match migrate lic function to what is in isolate script

While running the conversion script for the cp4ba conversion test, ran into an issue with the control namespace variable not being set in the `migrate_lic_cms` function. I have changed the function to mirror what is in the isolate.sh script. 

I have not tested this script change yet.

Testing:
- setup shared environment
- run conversion script
- verify script completes successfully, look specifically for the success message: `"Licensing configmaps copied from $namespace to $CONTROL_NS"` or info message: `"No licensing configmaps to migrate"`
